node/src/node_configurator/mod.rs

```diff
@@ -419,34 +232,6 @@ where
     }
 }
 
-// require two matching entries
-pub fn request_password_with_confirmation<F>(
-    confirmation_prompt: &str,
-    mismatch_msg: &str,
-    streams: &mut StdStreams,
-    verifier: F,
-) -> Result<String, PasswordError>
-where
-    F: FnOnce(&str) -> Result<(), String>,
-{
-    let reader_opt = possible_reader_from_stream(streams);
-    let password = read_password_with_reader(reader_opt).expect("Fatal error");
-    match verifier(&password) {
-        Ok(_) => {
-            flushed_write(streams.stdout, confirmation_prompt);
-            let reader_opt = possible_reader_from_stream(streams);
-            let confirm = read_password_with_reader(reader_opt).expect("Fatal error");
-            if password == confirm {
-                Ok(password)
-            } else {
-                flushed_write(streams.stdout, mismatch_msg);
-                Err(PasswordError::Mismatch)
-            }
-        }
-        Err(msg) => Err(PasswordError::VerifyError(msg)),
-    }
-}
-
 pub fn request_password_with_retry<R>(
```

Comment:
I'm afraid that if you decide that request_existing_password should go away then this function gets the same destiny.